crates/router/src/types/transformers.rs

```diff
@@ -619,3 +620,21 @@ impl TryFrom<domain::MerchantConnectorAccount> for api_models::admin::MerchantCo
         })
     }
 }
+
+impl ForeignFrom<storage_models::enums::CardNetwork> for api_models::enums::CardNetwork {
+    fn foreign_from(source: storage_models::enums::CardNetwork) -> Self {
+        match source {
+            storage_models::enums::CardNetwork::Visa => Self::Visa,
+            storage_models::enums::CardNetwork::Mastercard => Self::Mastercard,
+            storage_models::enums::CardNetwork::AmericanExpress => Self::AmericanExpress,
+            storage_models::enums::CardNetwork::JCB => Self::JCB,
+            storage_models::enums::CardNetwork::DinersClub => Self::DinersClub,
+            storage_models::enums::CardNetwork::Discover => Self::Discover,
+            storage_models::enums::CardNetwork::CartesBancaires => Self::CartesBancaires,
+            storage_models::enums::CardNetwork::UnionPay => Self::UnionPay,
+            storage_models::enums::CardNetwork::Interac => Self::Interac,
+            storage_models::enums::CardNetwork::RuPay => Self::RuPay,
+            storage_models::enums::CardNetwork::Maestro => Self::Maestro,
+        }
```

Comment:
This conversion can be done by the frunk macro. Please have a look at how it has been done for other enums in the https://github.com/juspay/hyperswitch/blob/fa392c40a86b2589a55c3adf1de5b862a544dbe9/crates/router/src/types/transformers.rs file

Reply:
done